oak/server/enclave_server.h

```diff
@@ -105,34 +110,29 @@ class EnclaveServer final : public asylo::TrustedApplication {
   asylo::StatusOr<std::unique_ptr<::grpc::Server>> CreateServer()
       EXCLUSIVE_LOCKS_REQUIRED(server_mutex_) {
     ::grpc::ServerBuilder builder;
-    // Bind address should be localhost + assigned port number.
+    // Bind address should be localhost + specified port number.
```

Comment:
If the bind address is localhost, remote hosts won't be able to connect?

Reply:
Thanks! Should have said "IPv6 any" address instead of localhost, but this is obsolete now